skeleton-common/usr/local/sbin/oc-fetch-ssh-keys

```diff
@@ -10,8 +10,8 @@ then
     chmod 700 /root/.ssh
 fi
 
-if [ $1 == "--upgrade" ]; then
-	/usr/local/bin/oc-metadata > /dev/null
+if [ $# == 1 ] && [ $1 == "--upgrade" ]; then
```

Comment:
if [ "$1" = "--upgrade" ]; then

(if $1 is not defined, it will compare "" = "--upgrade", which is what you want to do.)